### DISCUSSION TITLE
How can I edit custom style layers without using the built-in settings?

### DISCUSSION THREAD

**Discussion by Spekel** [dialogic-godot/dialogic#2431] [General]
Hello,

I am trying to use the custom styles feature to customize my text box. I was following along with [this video](https://www.youtube.com/watch?v=TLnzSzqBwu4). I made a new visual novel style then made the visual novel textbox layer custom. However, whenever I edit the nodes of the custom scene, such as replacing the theme override of the DialogTextPanel, the changes do not appear. They revert to the exported settings in the inspector. Per the video, the solution seems to be disabling the "Apply overrides on ready" setting, but this did not fix the issue. All of the exported "vn_textbox_layer.gd" settings are still being applied and overriding my custom colors, themes, etc that I am setting on the control nodes themselves. Anyone using custom layout scenes who can show me the correct workflow or know what is wrong?

Thank you

EDIT: I think I found the issue. The DialogicLayoutLayer class does not actually check the setting before calling _apply_export_overrides()

**Comment by AlleyTerry** [comment]
Hi I had the same issue were you able to figure out a fix?

**Reply by Spekel** [reply]
I just decided that I have to use the exported settings only for those properties for now. If you are using a fully custom style, not just one custom layer, then you could go into the dialogic_layout_layer.gd file and change the apply_export_overrides() function. I tried changing the last line to 'if apply_overrides_on_ready: _apply_export_overrides()'. Basically what it's doing is preventing any of the style tab settings from being applied ever.

**Reply by AlleyTerry** [reply]
That seemed to work thank you so much!
